pkg/cloud/services/eks/nodegroup.go

```diff
@@ -150,7 +150,7 @@ func (s *NodegroupService) remoteAccess() (*eks.RemoteAccessConfig, error) {
 	// SourceSecurityGroups is validated to be empty if PublicAccess is true
 	// but just in case we use an empty list to take advantage of the documented
 	// API behavior
-	var sSGs = []string{}
+	sSGs := []string{}
```

Comment:
Until here, there were quite a few minor, unneeded changes, plus some small improvements, both out of scope for the PR. If you can put the relevant ones into a separate PR and ping me, I'll get them in. Let's please avoid making this already-large PR review slower by this extra content.

Reply:
Sorry these were auto-linted, I missed removing them from the PR, I'll clean those up